src/integrationTest/groovy/wooga/gradle/unity/BaseBatchModeIntegrationSpec.groovy

```diff
@@ -179,6 +176,43 @@ class BaseBatchModeIntegrationSpec extends UnityIntegrationSpec {
         "redirectStdOut" | Unity         | 'false'              | 'true'              | "-logFile"                | true
         "redirectStdOut" | UnityPackage  | 'false'              | 'true'              | "-logFile"                | true
     }
+
+    @Unroll
+    def "set logCategory by task to #value with getter #useGetter"() {
+        given: ""
+        buildFile << """
+            task (mUnity, type: ${Unity.name}) {
+
+            }
+        """.stripIndent()
+
+        and:
+        if (useGetter) {
+            buildFile << """
+            mUnity.setLogCategory("${value}")
+        """.stripIndent()
+        } else {
+            buildFile << """
+            mUnity.logCategory = "${value}"
+        """.stripIndent()
+        }
```

Comment:
I would generate the method call. A `mUnity.logCategory = ...` is the same as `mUnity.setLogCategory(...)`
Check this test

https://github.com/wooga/atlas-unity/blob/b5fd4031f5bf44bcbc75da86b9cfb0fdbfaa2ef0/src/integrationTest/groovy/wooga/gradle/unity/UnityBatchModeIntegrationSpec.groovy#L21-L53